fw-child/resources/scss/child.scss

```diff
@@ -968,18 +970,25 @@ $query-card-hover-padding: 0.75rem;
       }
     }
 
-    &:hover {
-      .card:not(.variable-item) {
-        height: calc(100% + ($query-card-hover-padding * 2));
-        margin-top: $query-card-hover-padding * -1;
-        margin-left: $query-card-hover-padding * -1;
-        margin-right: $query-card-hover-padding * -1;
-        box-shadow: 0 0.25rem 1.5rem transparentize($black, 0.9);
-
-        .card-body {
-          margin-left: $query-card-hover-padding;
-          margin-right: $query-card-hover-padding;
-        }
+    &:hover .card:not(.no-resize),
+    .card.card-hover {
```

Comment:
The cards in the variables page have an hover effect that is different than the effect made by this `.card-hover` class. So the name `.card-hover` could create confusion as to why it refers to "hover" but it's not the effect of the hover for variables.

I suggest to use the name `.card-enlarged`.